### ISSUE TITLE
Cleanup of ctpps_2021 era

### ISSUE THREAD

**Issue by forthommel** [cms-sw/cmssw#38887]
As noted by @civanch in the review of #38280 the `ctpps_2021` era is to be superseded by `ctpps_2022` for obvious reasons. A systematic removal of all instances of the earlier in all parts of released and user-consumer code is to be performed as soon as the merging of #38280 is effective.

**Comment by makortel** [comment]
assign simulation,reconstruction,daq,operations

(based on packages shown by `git grep ctpps_2021`)

**Comment by forthommel** [comment]
This issue can be safely closed once #38280 is merged.

**Comment by forthommel** [comment]
With the merge of this PPS direct simulation-as-sequence PR the 2021 era was dropped and replaced by a 2022 era in all consumer codes. This issue can be closed.

**Comment by forthommel** [comment]
My bad, thanks @makortel :) Seems that only the GitHub Android app is not allowing it...
